$NetBSD: patch-configure,v 1.1 2012/10/24 08:33:51 manu Exp $

Make sure we do not link against -lpthread, as we only need mutexes
that can be found in libc.

Linking with -lpthread hurst with opensc-pkcs11.so since NetBSD-6.0
libpthread cannot be loaded by dlopen()

--- configure.orig	2012-10-19 11:29:36.000000000 +0200
+++ configure	2012-10-19 14:57:03.000000000 +0200
@@ -12821,8 +12821,17 @@
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 acx_pthread_ok=no
 
+case "${host_cpu}-${host_os}" in
+        *netbsd*)
+		acx_pthread_ok=yes
+		PTHREAD_CFLAGS=""
+		PTHREAD_LIBS=""
+		LIBS="$LIBS -lltdl"
+	;;
+esac
+
 # We used to check for pthread.h first, but this fails if pthread.h
 # requires special compiler flags (e.g. on True64 or Sequent).
 # It gets checked for in the link test anyway.
 
